### PR TITLE
Feat/implement alb

### PR DESCRIPTION
**Pull request by rmocanu-ionos**

### CODE REVIEW

**Review by gabisavu-ionos — APPROVED** [review]

**Review by avirtopeanu-ionos — APPROVED** [review]
Looks great! :+1:

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -97,6 +97,8 @@ jobs:
     strategy:
       matrix:
         include:
+          - namespace: application-loadbalancers
+            keywords: "application_load_balancer, target_group"
```
Comment (mimihalescu) [inline .github/workflows/CI.yml:101]: I would add here maybe `application-load-balancer` too, to match the test file names

File: tests/test.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+- hosts: localhost
```
Comment (mimihalescu) [inline tests/test.yml:1]: I think this file should be deleted

File: .github/workflows/CI.yml (hunk 1 above)
Comment (mimihalescu) [reply .github/workflows/CI.yml:101]: Same for `target-group`

**Review by mimihalescu — APPROVED** [review]

**Pull request merged**